pkg/model/datacatalog/api.go

```diff
@@ -42,3 +42,15 @@ type CreateAssetResponse struct {
 	// The ID of the created asset based on the source asset information given in CreateAssetRequest object
 	AssetID string `json:"assetID"`
 }
+
+type DeleteAssetRequest struct {
+	// The destination catalog id in which the asset to be deleted exists
+	DestinationCatalogID string `json:"destinationCatalogID"`
```

Comment:
again, why not `AssetID taxonomy.AssetID` ? 
@shlomitk1  ?

Reply:
Yes, should be `AssetID taxonomy.AssetID `json:"assetID"`` as in GetAssetRequest.

Reply:
Fixed. Thanks.